index.html

```diff
@@ -2247,6 +2091,175 @@
+[=cross-context recognition=] (and always [=recognizes=] across [=partitions=]); and
+[=same-site recognition=] is sometimes [=cross-context recognition=] (and may or may not
+involve multiple [=partitions=]).
+
+### User agent awareness of recognition {#user-agent-recognition}
+
+A <dfn>partition</dfn> is the [=user agent=]'s attempt to match how its user
+would understand a [=context=]. [=User agents=] don't have a perfect
+understanding of how their users experience the sites they visit, so they
+often need to approximate the boundaries between [=contexts=] when building
+[=partitions=].
+
+In the absence of better information, a [=partition=] can be defined as:
+
+* a set of [=environments=] (roughly: same-site and cross-site [^iframe^]s,
+workers, and top-level pages)
+* whose [=environment/top-level origins=] are in the [=same site=] (note: see
+[[PSL-Problems]])
+* being visited within the same user agent installation (and browser profile,
+container, or container tab for user agents that support those features)
+* between points in time that the person or user agent clears that [=site=]'s
+cookies and other storage (which is sometimes automatic at the end of each
+session).
+
+It can be difficult for a [=user agent=] to detect when a single site contains
+multiple [=contexts=]. When a [=user agent=] can detect this, it should adjust
+its [=partitions=] accordingly, for instance by partitioning identities
+per subdomain or site path. User agents should work to improve their ability to
+distinguish contexts within a site.
+
+[=User agents=] should prevent people from being [=cross-context
+recognition|recognized=] across [=partitions=] unless they intend to be
+recognized.
+
+Note that sites can do harm even if they can't be completely certain
+that visits come from the same person, so [=user agents=] should also take steps
+to prevent such probabilistic recognition. The [[[Privacy-Threat]]] discusses
+the tradeoffs involved ([[Privacy-Threat]]).
+
+If a [=user agent=] can tell that its user is using a particular identity on a
+website, it should make that active identity clear to the user (e.g. if the
+user logged into the site via an API like [[[credential-management-1]]]).
+
+### Recognition Methods {#recognition-methods}
+
+The web platform offers many ways for a website to recognize that a [=person=]
+is using the same [=identity=] over time, including [[RFC6265|cookies]],
+{{WindowLocalStorage/localStorage}}, {{WindowOrWorkerGlobalScope/indexedDB}},
+{{CacheStorage}}, and other forms of storage.
+
+Sometimes sites use this to save the [=person=]'s preferences or ongoing activities
+(e.g. shopping carts), and people have come to expect this behavior in some
+[=contexts=].
+
+People are unlikely to expect [=recognition=] all the time, and can find it
+difficult to prevent or mitigate, especially if it is automated.
+
+[=Recognition=] can be automated in different ways:
```

Comment:
Automation isn't quite the issue here, since first-party cookies are also automated. Maybe it's just when recognition uses ["unsanctioned"](https://www.w3.org/2001/tag/doc/unsanctioned-tracking/) methods, which we've extended to mean 3p cookies, link decoration, etc? I don't have a great idea for rewording, and you didn't add this concept, so I'm just noting the issue. We don't have to fix it to merge this PR.